tests/testthat/test-epinowcast.R

```diff
@@ -59,6 +59,20 @@ test_that("epinowcast() runs using default arguments only", {
   expect_error(nowcast$fit[[1]]$summary("rep_beta"))
 })
 
+test_that("epinowcast() runs with within-chain parallelisation", {
+  skip_on_cran()
+  skip_on_local()
+  obs <- germany_covid19_hosp[age_group %in% "00+"][location %in% "DE"] |>
+    enw_filter_report_dates(remove_days = 10) |>
+    enw_filter_reference_dates(include_days = 10)
+  pobs <- enw_preprocess_data(obs, max_delay = 5)
+  nowcast <- suppressMessages(epinowcast(pobs))
```

Comment:
don't we need to increase the number of threads_per_chain for this to have the desired effect?